common/ownership_voucher/ownership_voucher.go

```diff
@@ -0,0 +1,92 @@
+// Package ownershipvoucher provides helper functions for generating, parsing and verifying ownership vouchers.
+package ownershipvoucher
```

Comment:
Please add a comment to document the purpose of the package formatted:
// ownershipvoucher ...

Reply:
Done.